.gitignore

```diff
@@ -0,0 +1,8 @@
+/lib/helpers/secrets.py
```

Comment:
Usually this file would be present in the repo but secrets would be accessed via env variables.

Reply:
That's good to know. Can you help me set it up the right way ?